src/explorer.api/Models/NumericOverview.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace Explorer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+
+    using Aircloak.JsonApi;
+    using Aircloak.JsonApi.ResponseTypes;
+    using Explorer.Api.Models;
+
+    internal class NumericOverview : ColumnExplorer
+    {
+        internal NumericOverview(JsonApiSession apiSession, ExploreParams exploreParams)
+            : base(apiSession, exploreParams)
+        {
+        }
+
+        internal async IAsyncEnumerable<ExploreResult> Explore()
+        {
+            var queryParams = new QuerySpec(ExploreParams);
+
+            yield return new ExploreResult(ExplorationGuid, status: "waiting");
+
+            var queryResult = await ApiSession.Query<ResultRow>(
+                ExploreParams.DataSourceName,
+                queryParams.QueryStatement,
+                TimeSpan.FromMinutes(2));
+
+            var rows = queryResult.ResultRows;
+            Debug.Assert(
+                rows.Count() == 1,
+                $"Expected query {queryParams.QueryStatement} to return exactly one row.");
+
+            var stats = rows.First();
+
+            yield return new ExploreResult(ExplorationGuid, status: "complete", metrics: new List<ExploreResult.Metric>
+            {
+                new ExploreResult.Metric("Min")
+                {
+                    MetricType = AircloakType.Real,
+                    MetricValue = stats.Min,
+                },
+                new ExploreResult.Metric("Max")
+                {
+                    MetricType = AircloakType.Real,
+                    MetricValue = stats.Max,
+                },
+                new ExploreResult.Metric("Count")
+                {
+                    MetricType = AircloakType.Integer,
+                    MetricValue = stats.Count,
+                },
+                new ExploreResult.Metric("CountNoise")
+                {
+                    MetricType = AircloakType.Real,
+                    MetricValue = stats.CountNoise,
+                },
+            });
+        }
+
+        private class QuerySpec : IQuerySpec<ResultRow>
```

Comment:
Is this class related just to the NumericOverview? At a first glance it seems to be somewhat more general.

Reply:
Yes, this class defines the query specifically for the NumericOverview so i decided to define it within that class.

In retrospect it could be used elsewhere too so maybe it would be better to extract it (and the `ResultRow` class) into a `Explorer.Queries` namespace or something like that...

I wasn't sure how best to organise this so am open to suggestions.